client/src/app/containers/Home/active_game_list.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+import { selectCurrentUser, selectActiveGames } from '../../../auth/selectors';
+import { authSaga } from '../../../auth/saga';
+import { sliceKey, reducer, actions } from '../../../auth/slice';
+
+const ActiveGames = () => {
+  useInjectReducer({ key: sliceKey, reducer: reducer });
+  useInjectSaga({ key: sliceKey, saga: authSaga });
+  const dispatch = useDispatch();
+  const currentUser = useSelector(selectCurrentUser);
+  const activeGames = useSelector(selectActiveGames);
+  
+  // event handlers
+  const onRefreshActiveGames = () => {
+    dispatch(actions.refreshActiveGames(currentUser.id));
+  };
+
+  let gameButtons = [];
+  if ( activeGames ) {
+    for (const [gID, desc] of Object.entries(activeGames)) {
+        if ( !gID || !desc ) {
+            continue;
+        }
+        gameButtons.push(<br></br>)
+        gameButtons.push(<button
+            key={gID}
```

Comment:
If possible, we should sort these in some way (perhaps by gameID today) otherwise, the map iteration will make these ordered differently every time.